src/api/buildUrl.ts

```diff
@@ -0,0 +1,7 @@
+const BASE_API_URL = "https://api.offlinequran.com/api";
```

Comment:
Should be env var

Reply:
It can be, thing is, this is a default that doesn't change, and it's not a "secret", so there isn't really a good advantage to doing this no?

Reply:
Yeah it's a bit premature, we might not have another backend for some time